readme.md

```diff
@@ -116,6 +116,11 @@ Open DevTools for the specified `BrowserWindow` instance or the focused one.
 Type: `BrowserWindow`\
 Default: The focused `BrowserWindow`
 
+### preloadScript
```

Comment:
```suggestion
### preloadScriptPath
```

Reply:
https://github.com/sindresorhus/electron-debug#activates-devtools-extensions should also mention using this option when node integration is off.